tests/Fixtures/app/config/config_common.yml

```diff
@@ -36,6 +36,10 @@ api_platform:
         html:                          ['text/html']
         graphql:                       ['application/graphql']
         multipart:                     ['multipart/form-data']
+    docs_formats:
+        html:                          ['text/html']
+        jsonld:                        ['application/ld+json']
+        graphql:                       ['application/graphql']
```

Comment:
IMHO `graphql` is not a valid format for docs, as we already have another tool for it